mk_verificator/tests/rabbit_mq/test_rabbit_cluster.py

```diff
@@ -1 +1,43 @@
-__author__ = 'msenin'
+import json
+import re
+
+
+def test_checking_rabbitmq_cluster(local_salt_client):
+    # request pillar data from rmq nodes
+    rabbitmq_pillar_data = local_salt_client.cmd(
+        'rmq*',
+        'cmd.run',
+        ['salt-call pillar.data --output json rabbitmq:cluster'])
+
+    # creating dictionary {node:cluster_size_for_the_node}
```

Comment:
that's really good! but one more thing, may you please use parametrise decorator for this test and add also murano node:
 +@pytest.mark.parametrize(
+    "group",
+    utils.get_groups(utils.get_configuration(__file__))

because we should also check murano rabbit cluster as well

thanks!